frontend/src/packages/auth/auth-api.package.ts

```diff
@@ -67,6 +72,37 @@ class AuthApi extends HttpApi {
 
     return await response.json<UserAuthResponseDto>();
   }
+
+  public async sendEmailResetPasswordLink(
+    payload: AuthRequestPasswordDto,
+  ): Promise<SendEmailResponse> {
+    const response = await this.load(
+      this.getFullEndpoint(AuthApiPath.FORGOTTEN_PASSWORD, {}),
+      {
+        method: 'POST',
+        contentType: ContentType.JSON,
+        payload: JSON.stringify(payload),
+        hasAuth: false,
+      },
+    );
+
+    return await response.json<SendEmailResponse>();
+  }
```

Comment:
Please add empty line below

Reply:
Below return ? But every method in every package looks the same 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/44391531/1b80b51a-e3ba-4b59-a95e-7c29bdfc264a)

Reply:
This file `frontend/src/packages/auth/auth-api.package.ts` 🙂

Reply:
Yes , and as I said every other file looks the same

Reply:
I mean after the line 90